components/symbl_ai/symbl_ai.app.mjs

```diff
@@ -40,5 +40,11 @@ export default {
         data,
       });
     },
+    async getJobStatus(jobId) {
+      return this.makeRequest({
+        method: "get",
```

Comment:
It's not necessary to specify the method here since by default it's a `get` so you can remove it
```suggestion
```